### ISSUE TITLE
Search word that include % does not work correctly

### ISSUE THREAD

**Issue by kmdtmyk** [mrkamel/search_cop#65]
For example, items table has this records.

```sql
SELECT * FROM items;
```

|id|name|
|--|------|
|1|The 100% complete guide for Ruby on Rails|
|2|100G byte SSD|

```ruby
class Item < ApplicationRecord
  include SearchCop

  search_scope :search do
    attributes :name
  end

end

Item.search('100%')
```

I expected this returns a record with id 1 only. But, this returns both records.

**Comment by mrkamel** [comment]
thx for reporting. i agree. i'll change it.

**Comment by mrkamel** [comment]
available in 1.2.0
